docs/release-notes/highlights-1.7.0.asciidoc

```diff
@@ -42,5 +42,11 @@ After installing ECK 1.7.0 on Kubernetes versions 1.16/1.17 using Helm or the YA
 error: SchemaError(co.elastic.k8s.elasticsearch.v1.Elasticsearch.spec.nodeSets.volumeClaimTemplates): array should have exactly one sub-item
 ----
 
+This is due to a validation issue in `kubectl` that has been addressed in Kubernetes as of versions 1.18.12, 1.19.5 and 1.20. To work around this issue patch the Elasticsearch CRD as follows:
```

Comment:
I'm wondering how we could make it cleat that the fix is not in the client, but in the Kubernetes API server:

```suggestion
This is due to a validation issue in `kubectl` that has been addressed in the Kubernetes API server as of versions 1.18.13, 1.19.5 and 1.20. To work around this issue patch the Elasticsearch CRD as follows:
```

My fear is that some users may wrongly assume that if the issue is in `kubectl` then upgrading the client can fix the problem.

Reply:
Good point.